### ISSUE TITLE
Enable configuration of Checker Framework version

### ISSUE THREAD

**Issue by mernst** [jaredsburrows/gradle-checker-framework-plugin#16]
The plugin currently seems hard-coded to use version 2.4.0 of the Checker Framework.  Many bugs have been fixed, and features added, since then.

Please provide a configuration setting that I can use in my `build.gradle` file to specify what version of the Checker Framework the plugin will use.  (It would also be nice to change the default version to 2.6.0.)

**Comment by mernst** [comment]
Now the default should be changed to version 2.7.0.

(Is everything OK with the maintainers?  The project has gone quiet of late.)

**Comment by jaredsburrows** [comment]
Are you suggesting this be added to the extension?

**Comment by jaredsburrows** [comment]
Started working on this here - https://github.com/jaredsburrows/gradle-checker-framework-plugin/pull/19.

**Comment by mernst** [comment]
Thanks for the updates.
Your project has been forked by @kelloggm: https://github.com/kelloggm/checkerframework-gradle-plugin .
I have been successfully using his version, which [credits](https://github.com/kelloggm/checkerframework-gradle-plugin#credits) you for the original code.

Perhaps you could link to that version from your README, or join forces on that version.

**Comment by jaredsburrows** [comment]
@mernst It is not forked and my name is not in the git logs. Why would I link to code that was  copied my repository? Ideally, PR should have be sent here and submitted.

**Comment by mernst** [comment]
I speculate that the reason that no PR was submitted here is that you didn't respond to PRs and issues against this repository for two years.

**Comment by jaredsburrows** [comment]
Yes. Nice to see my name referenced in the README but my name is not in the git logs since it was copied and not forked.
